Authenticator/Source/OTPTokenListViewController.swift

```diff
@@ -200,7 +205,9 @@ extension OTPTokenListViewController /* UITableViewDataSource */ {
         moveRowAtIndexPath sourceIndexPath: NSIndexPath,
         toIndexPath destinationIndexPath: NSIndexPath)
     {
+        preventTableViewAnimations = true
```

Comment:
Opening Brace Spacing Violation: Opening braces should be preceded by a single space and on the same line as the declaration. (opening_brace)